de.tu_bs.cs.isf.e4cf.core/src/de/tu_bs/cs/isf/e4cf/core/util/tagging/ITagStore.java

```diff
@@ -1,33 +1,33 @@
-package de.tu_bs.cs.isf.e4cf.parts.project_explorer.tagging.store;
+package de.tu_bs.cs.isf.e4cf.core.util.tagging;
 
 import java.util.List;
 import java.util.Map;
 
-import de.tu_bs.cs.isf.e4cf.parts.project_explorer.tagging.Tag;
-
 public interface ITagStore {
 
 	/**
 	 * Load the available tags
-	 * @return available tags
+	 * @return loaded available tags
 	 */
 	public List<Tag> loadAvailableTags();
 	
 	/**
 	 * Store the available tags
-	 * @param availableTags
+	 * @param availableTags tags to store
 	 */
 	public void storeAvailableTags(List<Tag> availableTags);
 	
+
 	/**
 	 * Load a map of tags for each file
-	 * @return map with tags
+	 * @param availableTags because we only load existing tags
+	 * @return loaded map with tags
 	 */
 	public Map<String, List<Tag>> loadTagMap(List<Tag> availableTags);
 	
 	/**
 	 * Store a map of tags for each file
```

Comment:
Just from the interface javadoc it is still unclear what exactly these functions should do. E.g. what is the string part of this map.

As an interface documentation should definitely be more detailed.